examples/streamz/scripts/entry.sh

```diff
@@ -112,43 +109,22 @@ log "INFO" "Kafka and zookeeper running"
 $KAFKA_HOME/bin/kafka-topics.sh --create --bootstrap-server $broker --replication-factor 1 --partitions 1 --topic $input_topic
 $KAFKA_HOME/bin/kafka-topics.sh --create --bootstrap-server $broker --replication-factor 1 --partitions 1 --topic $output_topic
 log "INFO" "Kafka topics created"
+
 #**********************************
 # Read Sample Data
 #**********************************
 $KAFKA_HOME/bin/kafka-console-producer.sh --broker-list $broker --topic $input_topic < $data
 log "INFO" "Sample data read into kafka topic, $input_topic"
 
-#**********************************
```

Comment:
hmm is there any way we can keep this here? Only because starting dask this way was recommended as being more "production-like"... tpcx-bb also starts dask in a startup script I believe. Open to input